src/pages/researcher_console/DatasetSubmissionsTable.jsx

```diff
@@ -89,6 +130,24 @@ export default function DatasetSubmissionsTable(props) {
             Edit
           </Button>
         </div>;
+      const deleteButton = (status === 'Accepted') ?
+        <div/> :
+        <div>
+          <Link
+            style={{marginLeft: '15px'}}
+            id={`${term.datasetId}_delete`}
+            className={'glyphicon glyphicon-trash'}
+            onClick={() => handleClick(term)}
+            to={`#`}
+          />
+          <ConfirmationDialog 
+            title="Delete dataset" 
+            openState={open} 
+            close={handleClose}
+            action={() => removeDataset(selectedTerm)}
+            description={`Are you sure you want to delete the dataset named '${selectedTerm.datasetName}'?`} 
```

Comment:
Nit: I suggest using the dataset identifier here - names in production are extremely long and will affect the modal display here.